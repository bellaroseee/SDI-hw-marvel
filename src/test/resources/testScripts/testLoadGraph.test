# Test loading graphs with the created tsv files

LoadGraph dummy1 dummy1.tsv

LoadGraph dummy2 dummy2.tsv

LoadGraph dummy3 dummy3.tsv

LoadGraph dummy2 dummy3.tsv

# test empty data file

LoadGraph dummyE dummyEmpty.tsv