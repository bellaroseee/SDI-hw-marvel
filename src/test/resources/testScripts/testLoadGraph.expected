# Test loading graphs with the created tsv files



loaded graph dummy2

loaded graph dummy3


# test empty data file

loaded graph dummyE